region/maridia/inner-green/East Pants Room.json

```diff
@@ -214,6 +215,30 @@
         "It will float there until its next jump, where it will slowly descend."
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "Jump into Respawning Block",
+      "requires": [
+        "canSuitlessMaridia",
+        "HiJump",
+        "canJumpIntoRespawningBlock"
+      ]
+    },
+    {
+      "link": [1, 3],
+      "name": "SpringBall Jump into Respawning Block",
+      "requires": [
+        "canSuitlessMaridia",
+        "canTrickySpringBallJump",
+        "canTrickyJump",
+        "canJumpIntoRespawningBlock",
+        {"or": [
+          "can4HighMidAirMorph",
+          "canStationaryLateralMidAirMorph"
+        ]}
+      ],
+      "note": "It is recommended to shoot the ceiling block immediately upon room entry to bring one of the Puyos down to help once getting up there."
```

Comment:
```suggestion
      "note": "It is recommended to shoot the ceiling block immediately upon room entry to bring one of the Puyos down so that it will not be a problem later."
```